CMakeLists.txt

```diff
@@ -93,6 +93,15 @@ target_link_libraries(example_benchmark
   crazyflieLinkCpp
 )
 
+# example_broadcast
+add_executable(example_broadcast
+  examples/broadcast.cpp
+  examples/PacketUtils.hpp
```

Comment:
this should not be needed.

Reply:
Do you mean the `examples/PacketUtils.hpp` specifically, or this target as a whole?

Reply:
Just any *.h(pp) file doesn't need to declared in CMakeLists.txt